todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,66 @@
+import os
+from enum import Enum
+
+import pymongo
+from bson import ObjectId
+
+
+class ItemStatus(Enum):
+    TO_DO = 'TO_DO'
+    DONE = 'DONE'
```

Comment:
If you were using Python 3.11 or greater `StrEnum` may have been useful here (also could arguably be moved to its own file).